from src.logger.logger import logging
# from src.exception.exception import CustomException

# Create a logger

# logging.info("Log file testing")
print(logging.info("Log file testing second time"))

